### PR TITLE
Added no script warning

### PR DESCRIPTION
**Pull request by naveed-ahmad**
Fix for https://github.com/quran/quran.com-frontend/issues/526

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32998

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32999

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33000

**Comment by naveed-ahmad** [comment]
No script warning message.
noscript

Note fonts aren't loaded and player ins't working because javascript is disabled.

File: src/components/NoScriptWarning/style.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.noscript_warning {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 1201;
+  text-align: center;
+  font-weight: bold;
```
Comment (sabeurthabti) [inline src/components/NoScriptWarning/style.scss]: remove bold, looks a bit odd.  and ensure we use the base font-size (14px)
looks more natural:
screen shot 2016-12-13 at 23 17 37

File: src/components/NoScriptWarning/style.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  font-size: 120%;
```
Comment (sabeurthabti) [inline src/components/NoScriptWarning/style.scss]: Remove this so we fall back to the base font-size, which is 14px.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33002

**Comment by mmahalwy** [comment]
+1 on @sabeurthabti reviews

**Comment by mmahalwy** [comment]
thank you!

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33004

**Comment by mmahalwy** [comment]
@naveed-ahmad good to review and merge?

**Comment by naveed-ahmad** [comment]
@mmahalwy yep

**Pull request merged**